azavea/cac-tripplanner
# Feature/kak/remove relax category#956

## Overview

Per client request, the list of filter categories for destinations should *not* include 'relax'; we are not adding any other category in its place. This removes 'relax' from the filter UI and from the options in the admin site.


### Demo

![categories_admin_ui](https://user-images.githubusercontent.com/960264/34224710-d1ac1322-e591-11e7-8b0e-6bd3a3b82eff.png)

![categories_tabbar](https://user-images.githubusercontent.com/960264/34224721-d8e40992-e591-11e7-8990-1d7cf92ae451.png)

### Notes

Also adds a missing migration for watershed alliance, I think due to a change in the help text.


## Testing Instructions

 * `vagrant ssh app`
 * `cd /opt/app/python/cac_tripplanner/ && ./manage.py migrate`
 * Migrations should run okay
 * `cd /opt/app/src/ && npm run gulp-development`
 * Check out a destination/create a new destination: http://localhost:8024/admin/destinations/destination/
 * Category multi-select should not include 'relax'
 * Check out filter bar/dropdown on home page/map page; should have no 'relax' option


## Checklist


Closes #956
